admin/database/database.go

```diff
@@ -19,7 +19,7 @@ func Register(name string, driver Driver) {
 }
 
 // Open opens a new database connection
-func Open(driver string, dsn string) (DB, error) {
+func Open(driver, dsn string) (DB, error) {
```

Comment:
What lint rule is this? We should discuss, it might hurt readability

Reply:
But its a best practise most of the places i found, mostly everyone follow this rule, if we have same args type we should combine the type. I think it make sense to have this